### ISSUE TITLE
[FEATURE REQUEST] start/stop AC/BAT

### ISSUE THREAD

**Issue by Tuxino88** [jeffshee/gnome-ext-hanabi#20]
Great Work dude , congratulations !
Is it possible to add the start / stop option when connecting / disconnecting the charging cable?

**Comment by jeffshee** [comment]
Hi @Tuxino88,

I think that's a good idea, and I believe it is possible! 
Though there are still many core features that haven't been implemented yet. Might have to wait for a while.

**Comment by Tuxino88** [comment]
sure, take your time :D

little off topic question: Can you make the static wallpaper completely disappear? (you can see it behind)

**Comment by jeffshee** [comment]
>Can you make the static wallpaper completely disappear? (you can see it behind)

Yeah, I noticed that too. Unfortunately, it's quite tricky to hide that atm and might need more investigation. 😅

**Comment by jeffshee** [comment]
Hi @Tuxino88, this is implemented — Preferences now has **Pause on Battery** with a configurable **Low Battery Threshold**. Closing as completed (see also #94). Thanks for the suggestion! 🙏
